### ISSUE TITLE
Mini Player Standalone

### ISSUE THREAD

**Issue by ghost** [kmikiy/SpotMenu#35]
Could you also provide the option to show a standalone "mini player" like in this screenshot? => 

![bildschirmfoto 2017-07-17 15 55 03_99zyh8](https://user-images.githubusercontent.com/29753374/28271375-a09deaac-6b08-11e7-853e-b72552b8c132.jpg)

**Comment by ken89llo** [comment]
Looks good and minimal, especially on a "clean"  desktop 😆 . 
I think only the cover and hover over for controls is very nice, but should be a option (cover only and controls when hovered over/ cover and controls showed)

**Comment by kmikiy** [comment]
![ezgif-3-748c8dd2ed](https://user-images.githubusercontent.com/14206503/31332386-7d8ef04a-ace6-11e7-94ba-65094c3f1398.gif)
This is what I have so far

**Comment by fabi94music** [comment]
Looks awesome 👏!

**Comment by kmikiy** [comment]
how about this?
![ezgif com-optimize](https://user-images.githubusercontent.com/14206503/31415957-8220cf98-ae26-11e7-9836-a6de6c312f73.gif)

**Comment by fabi94music** [comment]
This looks great!! 🎉🎉

**Comment by fabi94music** [comment]
I like the option to move the MiniPlayer from your earlier post. Do you consider to add this ?
The new design is very nice btw

**Comment by kmikiy** [comment]
yep, I will add it but it’s not ready yet. I haven’t published an update in a while so I figured I should release what I had currently that works.
Glad you like the new design

**Comment by fabi94music** [comment]
Totally understand that :) And I am very happy with the new look. Only wish that the cover art of the notification center widget had also rounded edges just like the menu bar widget. :)

**Comment by ken89llo** [comment]
Yeah, do you have a early working version yet ? I’m excited 😊

**Comment by patrickrushton** [comment]
Any news on this? I would love to be able to "tear off" the artwork and have it float over my other apps the whole time.
